### PR TITLE
AddonAudio files

### PR DESCRIPTION
**Pull request by KacperBorowiecki**

### CODE REVIEW

File: G1ANT.Addon.AudioAddon/Addon.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    [Copyright(Author = "G1ANT Robot Ltd", Copyright = "G1ANT Robot Ltd", Email = "hi@g1ant.com", Website = "www.g1ant.com")]
+    [License(Type = "LGPL", ResourceName = "License.txt")]
+    //[CommandGroup(Name = "", Tooltip = "")]
+    public class Addon : Language.Addon
+    {
+
+        public override void Check()
+        {
+            base.Check();
+            // Check integrity of your Addon
```
Comment (pwitusik) [inline G1ANT.Addon.AudioAddon/Addon.cs]: remove comments

File: G1ANT.Addon.AudioAddon/G1ANT.Addon.AudioAddon.csproj [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    <Compile Include="Properties\AssemblyInfo.cs" />
+  </ItemGroup>
+  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <PropertyGroup>
+    <PreBuildEvent>
+    </PreBuildEvent>
+  </PropertyGroup>
+  <PropertyGroup>
+    <PostBuildEvent>
+      @ECHO OFF
```
Comment (pwitusik) [inline G1ANT.Addon.AudioAddon/G1ANT.Addon.AudioAddon.csproj]: remove post build event, it will fail on CI/CD

**Review by pwitusik — CHANGES_REQUESTED** [review]

File: G1ANT.Addon.AudioAddon/G1ANT.Addon.AudioAddon.csproj (hunk 2 above)
Comment (KacperBorowiecki) [reply G1ANT.Addon.AudioAddon/G1ANT.Addon.AudioAddon.csproj]: done

File: G1ANT.Addon.AudioAddon/Addon.cs (hunk 1 above)
Comment (KacperBorowiecki) [reply G1ANT.Addon.AudioAddon/Addon.cs]: done

**Review by pwitusik — APPROVED** [review]

**Pull request merged**